mongo/src/main/java/org/envirocar/server/mongo/convert/TrackSummariesConverter.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (C) 2013 The enviroCar project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.envirocar.server.mongo.convert;
+
+import org.envirocar.server.core.entities.DimensionedNumber;
+
+import com.github.jmkgreen.morphia.converters.SimpleValueConverter;
+import com.github.jmkgreen.morphia.converters.TypeConverter;
+import com.github.jmkgreen.morphia.mapping.MappedField;
+import com.github.jmkgreen.morphia.mapping.MappingException;
+import com.mongodb.BasicDBList;
+import com.mongodb.BasicDBObject;
+import com.mongodb.BasicDBObjectBuilder;
+import com.mongodb.DBObject;
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.geom.GeometryFactory;
+import java.util.ArrayList;
+import java.util.List;
+import org.bson.types.BasicBSONList;
+import org.envirocar.server.core.entities.TrackSummaries;
+import org.envirocar.server.core.entities.TrackSummary;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * MongoDB type converter for {@link DimensionedNumber}s.
+ *
+ * @author Maurin Radtke <maurin.radtke@uni-muenster.de>
+ */
+public class TrackSummariesConverter
```

Comment:
If you declare the `TrackSummaries`/`TrackSummary` as an `@Embedded` property, this should not be needed.